cmd/mailserver-canary/main.go

```diff
@@ -192,7 +192,7 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 		return nil, err
 	}
 
-	nodeConfig, err := params.NewNodeConfig(path.Join(workDir, ".ethereum"), "", params.FleetBeta, uint64(params.RopstenNetworkID))
+	nodeConfig, err := params.NewNodeConfig(path.Join(workDir, ".ethereum"), "", "", uint64(params.RopstenNetworkID))
```

Comment:
Maybe we can be more explicit with a const: `FleetNone = ""` or `FleetUndefined = ""` in https://github.com/status-im/status-go/blob/develop/params/cluster.go#L6-L9?